src/components/PostEditor.js

```diff
@@ -110,9 +111,8 @@ export class PostEditor extends React.PureComponent {
 
   componentDidMount () {
     // initialize the communities list when opening the editor in a community
-    const { community, postEdit: { communities }, tag } = this.props
-    if (community && isEmpty(communities)) this.addCommunity(community)
-    if (tag) this.updateStore({tag})
```

Comment:
This can't get dropped -- it's how we pre-set the tag when you open the post editor from a tag page.